components/src/dynamo/vllm/tests/test_vllm_instrumented_scheduler.py

```
@@ -2885,5 +2901,308 @@ def test_zero_request_decode_injection_is_skipped_immediately():
+        return stub
+
+    # rank0 timed out with only the admission FPM; rank1 got both.
+    stub0 = build(0, rank0, [_steady_fpm(126, 2, rank=0)])
+    stub1 = build(1, rank1, [_steady_fpm(126, 2, rank=1), _steady_fpm(128, 2, rank=1)])
+
+    errors: dict[int, Exception] = {}
+
+    def save(rank, stub):
+        try:
+            InstrumentedScheduler._bench_save_current_point(stub)
+        except Exception as error:  # pragma: no cover - failure diagnostics
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)